invite/test/e2e.test.ts

```diff
@@ -233,17 +241,21 @@ describe('end-to-end', () => {
 
   describe('when a comment includes no macros', () => {
     it('ignores it', async done => {
+      jest.spyOn(InviteBot.prototype, 'tryInvite');
       await triggerWebhook(probot, 'issue_comment.created');
-      expect(await db('invites').select().first()).toBeUndefined();
+      expect(InviteBot.prototype.tryInvite).not.toBeCalled();
+
+      // The test will fail if any unexpected network requests occur.
       done();
     });
   });
 
   describe('when someone joins without a recorded invitation', () => {
     it('ignores it', async done => {
       await triggerWebhook(probot, 'organization.member_added');
-      expect(await db('invites').select().first()).toBeUndefined();
+
+      // The test will fail if any unexpected network requests occur.
```

Comment:
Supernit: could still make this explicit in the code by adding something like
```typescript
expect(nock.isDone()).toBe(true);
```
(since this is _the_ thing you want to assert in this test)

Reply:
That's not the thing. There aren't any nocks that I want to assert are done; I want to be sure that there aren't any _un-nocked requests_, which nock already throws an error on.